docs/plan/issue-930.md

```
@@ -0,0 +1,138 @@
+  tests/unit/services/aiUsageLogger.test.js \
+  tests/unit/services/auditService.test.js \
+  tests/unit/services/analyticsDashboardService.test.js \
+  --coverage \
+  --collectCoverageFrom="src/services/alertService.js" \
+  --collectCoverageFrom="src/services/aiUsageLogger.js" \
+  --collectCoverageFrom="src/services/auditService.js" \
+  --collectCoverageFrom="src/services/analyticsDashboardService.js"
+```
+
+## Commit
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)